assets/src/components/AssignmentTable.js

```diff
@@ -45,6 +46,12 @@ const styles = theme => ({
   },
   narrowCell: {
     width: '120px'
+  },
+  possiblePointsText: {
+    margin: 'auto',
+    display: 'inline-block',
+    paddingTop: '22px',
+    verticalAlign: 'middle'
```

Comment:
These changes reflected as this
![Screen Shot 2020-01-31 at 10 57 53 AM](https://user-images.githubusercontent.com/8579775/73553891-b4b6bc80-4418-11ea-8053-d3f430181a7e.png)

Reply:
@pushyamig, I'm seeing now that the up/down toggle in the input is really close to the `/ 25`. Can we add some padding there?